doc/user/content/support.md

```diff
@@ -7,34 +7,54 @@ menu:
     weight: 10
 ---
 
-## Getting help
+Whether you're building, scaling, or debugging, we're here to help.
 
-### Self-managed Materialize (Community Edition)
+{{< note >}}
+We're retiring access to Slack and Microsoft Teams workspaces. We've introduced the messenger app to streamline and scale how users engage with support — starting with AI-first assistance and seamless escalation paths when needed.
+{{< /note >}}
 
-For the Community Edition, join the [Materialize Slack
-community](https://materialize.com/s/chat) or [open a
-discussion](https://github.com/MaterializeInc/materialize/discussions). Slack is
-the best place to get timely help from Materialize employees and other users!
+## Materialize Support
 
-### Self-managed Materialize (Enterprise Edition)
+We offer multiple support pathways depending on your platform and plan:
 
-If you're an Enterprise customer, you can access Materialize support via a
-shared channel within your company's Slack or Microsoft Teams workspace.
+### Messenger App
 
-To file a support ticket:
+You can reach us directly from:
 
-1. Send a message to the shared channel describing your issue.
-2. Apply the `:ticket:` (🎫) emoji to your message.
-3. Click the **File a ticket** button that appears.
-4. Select the priority that corresponds to the severity of the issue you are
-   experiencing. Consult the table below for definitions of error severities.
-5. Materialize will respond to the ticket within the response time specified
-   in your support contract.
+- **materialize.com**
 
-{{< note >}}
-Materialize only guarantees response times if you follow the above steps to
-convert a message into a support ticket.
-{{< /note >}}
+Our messenger now features Matty, our AI-powered support agent. Matty can help
+answer questions, suggest documentation, and assist with troubleshooting. You
+can also submit a request to **Talk to a human**. Just look for the messenger
+widget on the bottom right.
+
+{{< tip >}}
+If you are an Enterprise customer with SLA support and are experiencing an
+**urgent** issue, you can use the [**Report Production
+Outage**](#report-production-outage-for-sla-supported-customers-only) option for
+fastest response.
+{{</ tip >}}
+
+### Email
+
+You can also email us your questions at `support@materialize.com` from your
+Materialize login email address.
+
+### Join the Materialize Slack community
+
+Connect with other users, share ideas, and get informal help from the [Materialize user community](https://materialize.com/s/chat) on Slack.
+
+## Report Production Outage (For SLA Supported Customers Only)
```

Comment:
I changed this to **Report Production Outage (...)**  instead of **Urgent Issues (...)**
Can change back.